ladybug/src/main/resources/application.properties

```diff
@@ -1,11 +1 @@
-# Spring Application log settings
-logging.level.root=DEBUG
-logging.level.org.frankframework=DEBUG
-logging.level.org.apache.cxf=WARN
-logging.level.org.apache.coyote=WARN
-logging.level.org.apache.tomcat=WARN
-logging.level.org.apache.catalina=INFO
-logging.level.org.apache.jasper=WARN
-
-
 spring.main.allow-bean-definition-overriding=true
```

Comment:
We can't move the last property to XML configuration, so we don't need this whole file anymore?

Reply:
No, these are spring properties, they have nothing to do with the Frank.

Reply:
Ok, thanks.